Sources/SwiftGodot/Core/ClassServices.swift

```diff
@@ -227,7 +227,8 @@ public struct PropInfo {
                         class_name: classNamePtr,
                         hint: UInt32 (hint.rawValue),
                         hint_string: hintStrPtr,
-                        usage: UInt32 (usage.rawValue))
+                        usage: UInt32 (usage.rawValue)
+                    )
```

Comment:
Is there a particular reason why the closing parens was moved here?

Reply:
Oh, just a personal code formatting habit that leaked in. I'd love to start using something like swiftformat on this project some day, we've got a lot of styles mixing here.

For now I'll revert this part because it doesn't belong in the PR.